net/prometheus/Portfile

```diff
@@ -0,0 +1,120 @@
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        prometheus prometheus 2.10.0 v
+github.tarball_from archive
+
+description         The Prometheus monitoring system and time series database
+
+long_description    Prometheus is an open-source systems monitoring and \
+                    alerting toolkit originally built at SoundCloud. Since \
+                    its inception in 2012, many companies and organizations \
+                    have adopted Prometheus, and the project has a very \
+                    active developer and user community.
+
+homepage            https://prometheus.io/
+
+platforms           darwin
+categories          net
+license             apache
+
+maintainers         {gmail.com:herbygillot @herbygillot} openmaintainer
+
+depends_build       port:curl \
+                    port:go
```

Comment:
Is there a reason you are not using the golang portgroup?

Reply:
It's because we're using the project's Makefile to build, which does some prometheus-specific things, as opposed to building this as a straightforward Go project.